packages/medusa/src/services/cart.ts

```diff
@@ -1058,6 +1058,7 @@ class CartService extends TransactionBaseService {
           "items.variant",
           "items.variant.product",
           "shipping_methods",
+          "shipping_methods.shipping_option",
```

Comment:
@olivermrbl in order to simplify the future update around the repo, I ve aligned the cart repo, meaning that since it now use the query builder instead of find, the eager relations are not loaded by default. So I ve just updated the places it was missing and removed eager from the entity